src/gis/components/Map.js

```diff
@@ -32,11 +25,29 @@ L.Icon.Default.mergeOptions({
   shadowUrl: require('leaflet/dist/images/marker-shadow.png'),
 });
 
-const LeafletDraw = props => {
+const MapContext = React.createContext();
+
+function geojsonToLayer(geojson, layer, options = {}) {
+  layer.clearLayers();
+  L.geoJson(geojson, options).eachLayer(l => l.addTo(layer));
+}
+
+function layerToGeoJSON(layers) {
+  const features = layers
+    .filter(layer => layer.toGeoJSON)
+    .map(layer => layer.toGeoJSON());
+  return {
+    type: 'FeatureCollection',
+    features,
+  };
+}
```

Comment:
question: Will these get reused? Do they belong in `utils`?

Reply:
@paulschreiber Maybe, not right now, when it gets reused I think we should move it to a common place